modelforge/potential/utils.py

```diff
@@ -1177,6 +1177,81 @@ def nondimensionalize_distances(self, distances: torch.Tensor) -> torch.Tensor:
         ) / self.radial_scale_factor
 
 
+class TensorNetRadialBasisFunction(PhysNetRadialBasisFunction):
+    def __init__(
+            self,
+            number_of_radial_basis_functions: int,
+            max_distance: unit.Quantity,
+            min_distance: unit.Quantity = 0.0 * unit.nanometer,
+            alpha: unit.Quantity = 1.0 * unit.angstrom,
+            dtype: torch.dtype = torch.float32,
+            trainable_centers_and_scale_factors: bool = False,
+    ):
+        super().__init__(
+            number_of_radial_basis_functions,
+            max_distance,
+            min_distance,
+            alpha,
+            dtype,
+            trainable_centers_and_scale_factors,
+        )
+
```

Comment:
You can remove this. As @ianmkenney mentioned, if you don't overwrite the __init__ method in the child class the __init__ of the super class is used

Reply:
Solved.